### ISSUE TITLE
Abandoned package

### ISSUE THREAD

**Issue by IanDelMar** [burlingtonbytes/WP-Compiler#3]
Package oyejorge/less.php is abandoned, you should avoid using it.

**Comment by gschoppe** [comment]
Agreed... apparently wikimedia has taken over support https://github.com/wikimedia/less.php and now offers sourcemap generation, too.
